app/build_databse.py

```diff
@@ -0,0 +1,53 @@
+from externalapi import all_news_feed,headlines_feed,sources_feed
+from models.allnewsmodel import AllNewsModel
+from models.headlinesmodel import HeadlineModel
+from models.sourcesmodel import SourceModel
+from config import db
```

Comment:
I am a bit confused as to how this file is actually used. Does the code end up executing when imported? This is quite a risky idea - instead it's always best to at least encapsulate things in a function that you have more control when you want to be executed.